packages/web-app/src/components/groupCard/index.tsx

```diff
@@ -0,0 +1,97 @@
+import styled from 'styled-components';
+import React, {SyntheticEvent} from 'react';
+
+import FallbackImg from '../../../../ui-components/src/assets/avatar-token.svg'
+
+export type CardGroupProps = {
+  groupName: string;
+  groupDescription: string;
+  groupImageUrl: string;
+  memberCount: number | string;
+  groupUSDValue: string;
+  bgWhite?: boolean;
+};
+
+export const CardGroup: React.FC<CardGroupProps> = ({
+  bgWhite = false,
+  ...props
+}) => {
+
+  return (
+    <Card data-testid="cardToken" bgWhite={bgWhite}>
+      <CoinDetailsWithImage>
```

Comment:
It doesn't makes sense to have a container call Coin.. and have information about groups inside. Maybe GroupDetailsWithImage and same for the child